scripts/create-kubestellar-demo-env.sh

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env bash
+
+# The formal instructions accompaning this script can be found at:
+# https://docs.kubestellar.io/release-0.23.1/direct/get-started/
+
+set -e
+
+echo -e "Checking that pre-req softwares are installed..."
+curl -s -o check_pre_req.sh https://raw.githubusercontent.com/kubestellar/kubestellar/v0.23.1/hack/check_pre_req.sh
+chmod +x check_pre_req.sh
+./check_pre_req.sh -V kflex ocm helm kubectl docker kind
+
+output=$(./check_pre_req.sh -V kflex ocm helm kubectl docker kind)
+
+if echo "$output" | grep -q "X"; then
```

Comment:
Can the exit code of that command be used instead? If not, then perhaps this is an issue to raise against that script.

Reply:
fixed

Reply:
I see no change. I still see the output being examined with `grep`.